### PR TITLE
Add how to load probability tables to existing models to spaCy docs

### PR DESCRIPTION
**Pull request by thomashacker**
## Description

This `PR` adds a small section to the `usage/saving_loading` site of the spaCy docs. It explains how to load a probability table from [spacy-lookups-data](https://github.com/explosion/spacy-lookups-data) into pre-trained spaCy models like `en_core_web_sm`. It also shows how to do the same when training models from scratch in the `config.cfg`.

It was brought up by a user in this discussion [#6388](https://github.com/explosion/spaCy/discussions/6388).

### Types of change

Change to documentation

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/docs/usage/saving-loading.md [hunk 1]
```diff
@@ -313,6 +313,29 @@ installed in the same environment – that's it.
...
+```python
+### load_table.py
+import spacy
+from spacy.lookups import load_lookups
+nlp = spacy.load("en_core_web_sm")
+lookups = load_lookups("en", ["lexeme_prob"])
+nlp.vocab.lookups.add_table("lexeme_prob", lookups.get_table("lexeme_prob"))
+```
+
+When training a model from scratch you can also specify probability tables in the `config.cfg`.
```
Comment (adrianeboyd) [inline website/docs/usage/saving-loading.md:329]: This description doesn't correspond to the example, which is for `lexeme_norm` instead of `lexeme_prob`.

**Comment by thomashacker** [comment]
Merged with master since #12005 was also merged and made sure to adhere to the new syntax changes.

File: website/docs/usage/saving-loading.md (hunk 1 above)
Comment (adrianeboyd) [reply website/docs/usage/saving-loading.md]: The probability tables are called `lexeme_prob`, not `lexeme_norm`. (Both exist and would work in these examples, but this example is intended for `lexeme_prob`, right?)

Comment (thomashacker) [reply website/docs/usage/saving-loading.md]: Oh sorry, I misunderstood the first message and thought I should rename to `_norm`. I'll change it back to `_prob`.

Comment (thomashacker) [reply website/docs/usage/saving-loading.md]: Changed to `lexeme_prob` and made sure that both examples (`.py` and `.cfg`) are consistent this time 😇

**Review by adrianeboyd — COMMENTED** [review]
Sorry, I'd accidentally left a review note as a draft...

**Comment by adrianeboyd** [comment]
Looks good!

**Pull request merged**